### DISCUSSION TITLE
Ellipsoid Reprsentation

### DISCUSSION THREAD

**Discussion by BradyAJohnston** [molstar/molstar#407] [Q&A]
Firstly, thanks for working on such a fantastic package! Having so much functionality in the browser, and it being so performant, is amazing!

Secondly, is it possible at all to create these kind of ellipsoid representations for atoms? I've tried the ellipsoid representation but it seems to be a stick-style representation. 

![](https://cdn.discordapp.com/attachments/945527499268440094/956371402234155048/Crystal-structure-of-23-Thermal-ellipsoids-are-represented-at-the-30-level-and-hydrogen.png)

**Comment by arose** [comment]
Thanks @BradyAJohnston! The ellipsoid representation is the correct one to use. You will need a file that has atom anisotropy data. Currently that is parsed (if available) from mmCIF (e.g., from PDB) and coreCIF (e.g., from CCDC, CSD, ...). Can you provide an example file where you expect it to work but it doesn't?

Here is an example from the PDB: [5D8V](https://www.rcsb.org/structure/5D8V)

![image](https://user-images.githubusercontent.com/272250/160730175-fb436586-b336-43dc-abb7-8ec3644b4554.png)
